README.md

```diff
@@ -20,25 +20,113 @@ repositories {
 }
 
 dependencies {
-    implementation "com.github.nextcloud:Android-SingleSignOn:0.3.0"
+	implementation 'com.google.code.gson:gson:2.8.5'
```

Comment:
If this is needed, it should be a dep from SSO

Reply:
The only thing I know is that the read me has this line of code in onActivityResult:

`NextcloudAPI nextcloudAPI = new NextcloudAPI(context, ssoAccount, new GsonBuilder().create(), callback);`

And if you don't include the implementation 'com.google.code.gson:gson:2.8.5' dependency it does not build.

Reply:
@tobiasKaminsky indeed.. would be better to include it as a dependency of the sso library as it is mandatory.

Reply:
Just opened #67 as a TODO to work on this.